src/towncrier/_project.py

```diff
@@ -82,37 +78,30 @@ def get_version(package_dir: str, package: str) -> str:
     if isinstance(version, str):
         return version.strip()
 
-    if isinstance(version, IncrementalVersion):
-        # FIXME:https://github.com/twisted/incremental/issues/81
-        # Incremental uses `.rcN`.
-        # importlib uses `rcN` (without a dot separation).
-        # Here we make incremental work like importlib.
-        return version.base().strip().replace(".rc", "rc")
-
     if isinstance(version, tuple):
         return ".".join(map(str, version)).strip()
 
+    # Try duck-typing as an Incremental version.
+    if hasattr(version, "base"):
+        try:
+            version = str(version.base()).strip()
+            # Incremental uses `X.Y.rcN`.
+            # Standardize on importlib (and PEP440) use of `X.YrcN`:
+            return version.replace(".rc", "rc")  # type: ignore
```

Comment:
I don't know why coverage is reporting this line as not covered.... we need to check

Reply:
There was an indentation problem with a test, fixed now